packages/ampd-sdk/src/main.rs

```diff
@@ -0,0 +1,3 @@
+fn main() {
```

Comment:
The SDK should be created with `cargo new .. --lib` so there will not be this main.rs